labelbox/schema/dataset.py

```diff
@@ -67,7 +67,7 @@ def create_data_row(self, **kwargs):
 
         return self.client._create(DataRow, kwargs)
 
-    def create_data_rows(self, items):
+    def create_data_rows(self, items, tiled_imagery=False):
```

Comment:
Once again not a big fan of continueing to add boolean arguments, could we just add another method that uploads `tiled_imagery` instead of overloading this one?

Reply:
yeah this is definitely a problematic boolean param because we will want to get rid of it at some point. I'll see if there is a reliable way to auto detect the type.

Reply:
Removed.